src/recoilState/appState.ts

```diff
@@ -91,6 +93,24 @@ export const rMyProfile = selector<IApiFilledProfile | undefined>({
 });
 export const useMyProfile = () => useRecoilValue(rMyProfile);
 
+export const rMyAvatar = selector<File>({
+  key: 'rMyAvatar',
+  get: async ({ get }) => {
+    const avatar = get(rMyProfile)?.avatar;
+    if (!avatar) {
+      return neverEndingPromise<File>();
```

Comment:
It will look like:
```
<Suspense
      fallback={
        <div className={classes.navLinks}>
          <Skeleton width={100} height={20} />
          <Skeleton width={100} height={20} />
        </div>
      }
    > <HeaderNav />
    </Suspense>
  ```
  
Showing fallback.
  
Depends which fallback get's triggered. I don't have a good way to debug these, it has rarely happened to me. I've used print statements - unfortunately the recoilize extension has been abandoned..